pyproject.toml

```diff
@@ -89,6 +91,7 @@ test = [
     "dowhy",
     "osqp<1.0.0,>=0.6.2",
     "pygraphviz",
+    "preliz>=0.8.0",
```

Comment:
@williambdean, probably we need to update to 0.20.0

Reply:
I think that is stale commit now